bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/handler/HaywardFilterHandler.java

```diff
@@ -45,7 +50,74 @@ public HaywardFilterHandler(Thing thing) {
 
     @Override
     public void initialize() {
-        updateStatus(ThingStatus.ONLINE);
+        try {
+            setStateDescriptions();
+            updateStatus(ThingStatus.ONLINE);
+        } catch (HaywardException e) {
+            updateStatus(ThingStatus.OFFLINE);
```

Comment:
Can you set the ThingStatusDetail and a message, so that the user knows where to start searching when this happens?